solidity/migrations/networkInfo.js

```diff
@@ -48,4 +48,9 @@ module.exports = {
     bitcoin: bitcoinTest,
     firstID: ID_SPACE_SIZE.muln(truffleConf.networks.alfajores_test.network_id + 0x800000)
   },
+  local_test: {
```

Comment:
What name have we used elsewhere for this? If I remember correctly truffle tends to (and we have followed this) use simply `local` for a local testnet.

Reply:
As Lukasz said, the script (2_deploy_contracts.js) in the maintainer look at the network name and if it ends with "_test", they deploy smart contracts for Bitcoin testnet (TestnetRelay), otherwise they deploy for mainnet (OnDemandSPV).